imap_processing/cli.py

```diff
@@ -406,10 +428,16 @@ def post_processing(self, datasets: list[xr.Dataset]) -> None:
             return
 
         logger.info("Writing products to local storage")
-        logger.info("Parent files: %s", self._dependency_list)
+
+        list_of_files = [
+            dep_ojb.filename_list
+            for dep_ojb in self._dependency_collection.processing_input
```

Comment:
dep_obj.filename_list
for dep_obj in self._dependency_collection.processing_input
```suggestion
            dep_obj.filename_list
            for dep_obj in self._dependency_collection.processing_input
```

Reply:
good catch! Sometimes my brain thinks faster than my finger can type and this happens...lol